src/queue.c

```diff
@@ -5762,7 +5762,25 @@ DISPATCH_NOINLINE DISPATCH_NORETURN
 static void
 _dispatch_sigsuspend(void)
 {
-	static const sigset_t mask;
+	sigset_t mask;
+	int r;
+
+	r = sigemptyset(&mask);
+	(void)dispatch_assume_zero(r);
+
+#if DISPATCH_EVENT_BACKEND_EPOLL
+	// If there are signals that the user may have reasonably
```

Comment:
that feels like the wrong solution, but I need to think about it more. I hate signals.